service/src/main/java/uk/gov/hmcts/reform/fpl/service/document/ManageDocumentService.java

```diff
@@ -74,7 +72,7 @@
         }
 
         //If toggle not on, replicate old behaviour (always show C2 list, even if empty)
```

Comment:
@toireasam I think you worked on this, do we still need to leave in the building of the dynamic list?

Reply:
my pr was only relating to documents in open state, this is pre submitted so @joez-f might know more, but yeah i think removing what you have done looks fine to me